services/core-web/src/components/navigation/NavBar.js

```diff
@@ -314,6 +314,18 @@ export class NavBar extends Component {
               </Link>
             </Col>
           </Row>
+          <Row>
+            <Col span={24}>
+              <Link to={router.MAJOR_PROJECTS_DASHBOARD.route}>
```

Comment:
Are we putting this dashboard behind a feature/ENV wall first for testing or is it straight to Prod?

Reply:
Good idea, going to put the behind feature flag